src/runtime/slot-dom-consumer.ts

```diff
@@ -21,7 +21,8 @@ export interface DomRendering extends Rendering {
   liveDom?: Template;
 }
 
-const templateByName = new Map();
+// tslint:disable-next-line: no-any
+const templateByName = new Map<string, any>();
```

Comment:
This is a map of `string` to `Template` (the latter is a DOM construct).

Reply:
Thanks